### PR TITLE
chore: Add FOSSA check to GH actions

### PR DESCRIPTION
**Pull request by ghislainbourgeois**

### CODE REVIEW

**Comment by gab-arrobo** [comment]
ok to test

File: .github/workflows/main.yml
Comment (gab-arrobo) [inline .github/workflows/main.yml:1]: This repo has a `reuse.yml` file to check for license. Given that you are adding the license check here, can you please remove `reuse.yml` as part of this PR?

**Review by gab-arrobo — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml
Comment (ghislainbourgeois) [reply .github/workflows/main.yml:1]: Done

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**